backend/routes/comments.js

```diff
@@ -45,7 +45,9 @@ router.post("/comments", async (req, res) => {
             StockId: stock.id
         };
 
-        const comment = await Comment.create(commentData);
+        const newComment = await Comment.create(commentData);
+
+        const comment = await Comment.findOne({ where: { id: newComment.id }, include: [{ model: User }] });
```

Comment:
interesting, what exactly was the issue with `comment` before?

Reply:
The newComment wouldn't include the user's information if I did include: [{model: User}] in the create function for some reason so I had to use findOne to solve that.